src/Validation/Validation.php

```diff
@@ -308,6 +308,35 @@ public static function compareWith($check, $field, $context)
     }
 
     /**
+     * Checks if a string contains one or more non-alphanumeric characters.
+     *
+     * Returns true if string contains at least the specified number of non-alphanumeric characters
+     *
+     * $check can be passed as an array:
+     * ['check' => 'valueToCheck'];
+     *
+     * @param string|array $check Value to check
+     * @param int $count Number of non-alphanumerics to check for
+     * @return bool Success
+     */
+    public static function containsNonAlphaNumeric($check, $count = 1)
+    {
+        if (is_array($check)) {
+            extract(static::_defaults($check));
+        }
+
+        if (empty($check) && $check != '0') {
```

Comment:
needs to be strict, but why the additional check? `'0'` is also covered by empty().